contrib/python-pytest-mock/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "python-pytest-mock"
+pkgver = "3.12.0"
+pkgrel = 0
+build_style = "python_pep517"
+
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+checkdepends = ["python-pytest-asyncio"]
+depends = ["python-mock", "python-pytest"]
+pkgdesc = "Thin-wrapper around the mock package for easier use with pytest"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MIT"
+url = "https://pytest-mock.readthedocs.io/en/latest/index.html"
+source = f"https://github.com/pytest-dev/pytest-mock/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "aeedddfe2feefde936ba3d723e26db5757570537ab66bf4dd2cd17d4c263f022"
+
+
+def init_build(self):
+    self.env["SETUPTOOLS_SCM_PRETEND_VERSION"] = f"{pkgver}"
```

Comment:
if you use the pypi source you don't need this

the only reason to use git directly is for when the pypi source has the tests removed from it, but it has them here

Reply:
okay